core/src/test/java/org/elasticsearch/index/translog/TranslogTests.java

```diff
@@ -1008,42 +1032,6 @@ public void testTranslogWriter() throws IOException {
         IOUtils.close(writer);
     }
 
-    public void testFailWriterWhileClosing() throws IOException {
```

Comment:
good that we have one thing less to worry about, but I wonder if it's a good idea to remove the test. I mean these transitions are always tricky and it's good to have it well tested?